test/ClangImporter/bad-ns-extensible-string-enum.swift

```diff
@@ -0,0 +1,3 @@
+// RUN: %target-swift-frontend -typecheck %s -import-objc-header %S/Inputs/bad-ns-extensible-string-enum.h -verify
```

Comment:
Two requests:
- Please add `// REQUIRES: objc_interop` to keep this from being run on non-ObjC-friendly platforms.
- Ideally, this would work with the "mock SDK" in test/Inputs/, which is a bit faster to build and doesn't depend on what's in the real SDK. We have a special syntax for that: `%target-swift-frontend(mock-sdk: %clang-importer-sdk)`. Does the test still pass with that?

Reply:
I don't get it where I have to put the string in the second bullet point.

And one thing, there is somewhere come kind of guide on how to use the comments for the tests? For this I was merely copying from other files but I prefer to know what I am doing and avoid this pitfalls

Reply:
Ah, right. There's some of it in [docs/Testing.md](https://github.com/apple/swift/blob/master/docs/Testing.md#writing-tests), but we should probably fill that out and organize it a little better. The format's the same as what LLVM uses, so the [LLVM Testing Guide](http://www.llvm.org/docs/TestingGuide.html#writing-new-regression-tests) is also a useful reference.

In this case, the string I put replaces your existing `%target-swift-frontend`.

Reply:
I have changed the first line of the comment with `RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -verify -I %S/Inputs/bad-ns-extensible-string-enum.h %s` but the compiler is crashing on me with "opening import file for module 'Swift': Not a directory"

I'm doing something wrong?

Reply:
`-I` provides a search path, rather than importing the header as a bridging header. You're looking for `-import-objc-header`.

Reply:
Now with this comment `RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck %s -import-objc-header %S/Inputs/bad-ns-extensible-string-enum.h -verify` it is failing with "failed to import bridging header"

Reply:
Okay, that's better. I suspect what's failing is the use of `NS_EXTENSIBLE_STRING_ENUM`, which we probably don't define anywhere in test/Inputs/clang-importer-sdk/. We could put a definition in there, but for now you might as well replace it with the attribute it expands to, `__attribute__((swift_wrapper(struct)))`.